ddtrace/context.py

```diff
@@ -142,8 +142,12 @@ def _traceparent(self):
         else:
             trace_id = "{:032x}".format(self.trace_id)
 
-        sampled = 1 if self.sampling_priority and self.sampling_priority > 0 else 0
-        return "00-{}-{:016x}-{:02x}".format(trace_id, self.span_id, sampled)
+        return "00-{}-{:016x}-{}".format(trace_id, self.span_id, self._traceflags)
+
+    @property
+    def _traceflags(self):
```

Comment:
We changed this because it's a bit cleaner/clearer and to allow for the easy addition of other flags in the future? https://www.w3.org/TR/trace-context/#other-flags

Reply:
This reverts the "future proofing" to make this code more readable. We can add support for other flags when w3c updates the spec.